src/main/java/com/exadel/movie_new/controller/MovieController.java

```diff
@@ -0,0 +1,48 @@
+package com.exadel.movie_new.controller;
+
+import com.exadel.movie_new.model.Movie;
+import com.exadel.movie_new.service.MovieService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+public class MovieController {
+
+    private MovieService movieService;
+    public MovieController(MovieService movieService) {
+        this.movieService = movieService;
+    }
+
+
+    @PostMapping("/addmovie")

```

Comment:
not exactly what i want. You must leave "/movie" or "/movies" (preferable) because it's common URL path. Usually such parts uses for all controllers. F.e UserController will have "/users" and so on. But you should not set "/movies" for all enpoints like @POST("/movies") etc. It's better to use it on the class level @RestController @Path("/movies"). And then, according to REST philosophy, each endpoint even should not have any additional mappings, because it's obviously if you are making: POST to "/movies" - you creating movie, PUT - update some movie, GET - get all movie, DELETE - delete one movie (by some parameter, usual it's ID), and finally GET("/{movieId}") - using movieId as pathParam allows you get only one particular movie(not all of them like in case GET without parameters). That's main idea - you Controller must be responcible only for one type of entities, in our case it's "movie". And MovieController should not do nothing with user, because for this you will have UserController, so URL for each controller must have clear description what each operation will do.